src/components/Modal/index.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+function Modal({ children }) {
+  return (
+    <div>
+      <Link to="..">
+        <div className="top-0 left-0 w-screen h-screen"></div>
+      </Link>
+      <div className="max-w-full overflow-y-auto transform -translate-x-1/2 -translate-y-1/2 rounded-lg ixed top-1/2 left-1/2 w-80 md:w-full max-h-90vh">
```

Comment:
fixed 속성을 주시고 싶으셨던게 맞을까요? 아래 오탈자 있는 것 같습니다! 확인 부탁드려요 :)

```suggestion
      <div className="max-w-full overflow-y-auto transform -translate-x-1/2 -translate-y-1/2 rounded-lg fixed top-1/2 left-1/2 w-80 md:w-full max-h-90vh">
```

Reply:
빠르게 수정하겠습니다!